### PR TITLE
Made end screen

### PR DESCRIPTION
**Pull request by KingPleather**
made end screen

### CODE REVIEW

File: src/screens/EndScreen.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { View, Image, Text } from "react-native";
+import Ionicons from "@expo/vector-icons/Ionicons";
```
Comment (TenType) [inline src/screens/EndScreen.js:2]: New dependency added, but looks like there were no `package.json` or `package-lock.json` files committed

Comment (TenType) [inline src/screens/EndScreen.js:2]: New dependency added, but looks like there were no `package.json` or `package-lock.json` files committed

**Review by TenType — CHANGES_REQUESTED** [review]
I don't think all of your changes were committed because the screen is not accessible anywhere in the app and there are missing dependency edits (see my other comment)

Also needs a few more things:
- [x] End screen should show up when the form is submitted
- [ ] There needs to be a way for the user to leave the end screen

File: src/screens/EndScreen.js (hunk 1 above)
Comment (KingPleather) [reply src/screens/EndScreen.js:2]: I dont think i did any installing, i just followed expo's instructions https://icons.expo.fyi/Index/FontAwesome/refresh something like this. Maybe I am wrong though, and if so, how do i add the changes cuz its not showing up in the github desktop.

Comment (TenType) [reply src/screens/EndScreen.js:2]: Never mind, the vector icons library is part of the [default expo package](https://docs.expo.dev/guides/icons/), so you're good

**Review by TenType — APPROVED** [review]
Further end screen improvements in #65

**Pull request merged**